utility/src/test/java/org/oppia/android/util/parser/image/UrlImageParserTest.kt

```diff
@@ -58,9 +60,6 @@ class UrlImageParserTest {
     )
   }
 
-  // TODO(#277): Add more test cases for loading images. The below doesn't include layout or
```

Comment:
I think we need to revert this TODO. This PR only fixes part of the issue, and todo should be removed in a PR expected to close the issue.